src/lib.rs

```diff
@@ -162,6 +163,7 @@ impl std::default::Default for BrowserOptions {
     fn default() -> Self {
         BrowserOptions {
             suppress_output: true,
+            target_hint: String::from("_blank"),
```

Comment:
Should be `option_env!("WEBBROWSER_WASM_TARGET")` to ensure back comptability.

Reply:
Oh right, fixed.